code/datums/diseases/advance/symptoms/vomit.dm

```diff
@@ -28,79 +28,31 @@ Bonus
 	transmittable = 1
 	level = 3
 	severity = 4
+	base_message_chance = 100
+	symptom_delay_min = 25
+	symptom_delay_max = 80
+	var/vomit_blood = FALSE
+	var/proj_vomit = FALSE

```

Comment:
Perhaps have this be an integer rather than a boolean, as it's something that can be scaled.